package.json

```diff
@@ -7,6 +7,11 @@
 		"url": "git://github.com/browserify/resolve.git"
 	},
 	"main": "index.js",
+	"files": [
+		"/lib",
+		"/index.js",
+		"/index.mjs"
+	],
```

Comment:
"files" is highly dangerous and should never be used; only `.npmignore` should be used to control what files are published.

Reply:
Highly disagree, but you're the boss here. 🧐

Reply:
"a lot of packages use it" isn't an argument for it being safe.

The worst possible damage caused by accidental omission from "npmignore" is "the package is slightly larger". The worst possible damage caused by accidental omission from "files" is "every consumer breaks".